sdk/internal/metadata/client.go

```diff
@@ -24,10 +25,25 @@ func NewClientWithEndpoint(endpoint string) *Client {
 	}
 }
 
-func (c *Client) GetMetaData(ctx context.Context) (*MetaData, error) {
-	metadata, err := c.getMetaDataFrom2022API(ctx)
+func (c *Client) GetMetaData(ctx context.Context, name string) (*MetaData, error) {
+	var metadata *metaDataResponse
+	var err error
+
+	// USGovernment does not yet support latest schema
+	version := "2022-09-01"
+	if strings.Contains(name, "USGovernment") {
```

Comment:
do we want to case-insensitively check this?

Reply:
Removed this in favour of automatic fallback on unmarshal errors